diff --git a/lib/FileSystemInfo.js b/lib/FileSystemInfo.js
index fce076798c795ba4d06516ddaf169764d97ad3dc..77c922298401f7f7dc2ec11e66b1bf949472d5c4 100644
--- a/lib/FileSystemInfo.js
+++ b/lib/FileSystemInfo.js
@@ -1479,7 +1479,7 @@ class FileSystemInfo {
 					resolveResults.set(key, undefined);
 					resolve(context, path, resolverContext, (err, _, result) => {
 						if (typeof expected === "string") {
-							if (!err && result && result.path === expected) {
+							if (!err && result && result.path.toLowerCase() === expected.toLowerCase()) {
 								resolveResults.set(key, result.path);
 							} else {
 								invalidResolveResults.add(key);